Kudu.Contracts/Jobs/IContinuousJobsManager.cs

```diff
@@ -1,9 +1,14 @@
-﻿namespace Kudu.Contracts.Jobs
+﻿using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace Kudu.Contracts.Jobs
 {
     public interface IContinuousJobsManager : IJobsManager<ContinuousJob>
     {
         void DisableJob(string jobName);
 
         void EnableJob(string jobName);
+
+        Task<HttpResponseMessage> RequestPassthrough(string jobName, string path, HttpRequestMessage request);
```

Comment:
Can you make this method name a verb - perhaps HandleRequest, ProcessRequest, etc.?